src/platform/src/main/java/org/geoserver/platform/resource/Paths.java

```diff
@@ -127,7 +161,11 @@ private static String toPath(List<String> names) {
                 throw new IllegalArgumentException("Contains invalid " + item + " path: " + buf);
             }
             if (!WARN.matcher(item).matches()) {
-                // to do warning 
+                if (STRICT_PATH == true) {
+                    throw new IllegalArgumentException("Contains invalid " + item + " path: " + buf);
+                } else {
+                    // TODO: warning
```

Comment:
We are not going to provide the warning as part of this pull request. We can do a second one ...

If I understand the discussion we could provide a "tip" under the name field indicating what protocols the name can be used with. So if the user provides a name with "," we can say "Style can be used as as a default style, but is incompatible with GetMap and GetFeatureInfo requests".